### PR TITLE
[bug fix]: print logs repeatedly

### PR DESCRIPTION
**Pull request by Yancey1989**
Fixed https://github.com/sql-machine-learning/sqlflow/issues/416

### CODE REVIEW

File: sqlflow/magic.py [hunk 1]
```diff
@@ -45,11 +45,6 @@ def execute(self, line, cell):
         return self.client.execute('\n'.join([line, cell]))
 
 def load_ipython_extension(ipython):
-    out_handler = logging.StreamHandler(sys.stdout)
-    out_handler.setLevel(logging.INFO)
-    _LOGGER.addHandler(out_handler)
-    _LOGGER.setLevel(logging.INFO)
-
```
Comment (tonyyang-svail) [inline sqlflow/magic.py]: @Yancey1989 Thanks for figuring out the cause for https://github.com/sql-machine-learning/sqlflow/issues/416!

File: sqlflow/client.py [hunk 2]
```diff
@@ -8,9 +9,10 @@
 import sqlflow.proto.sqlflow_pb2 as pb
 import sqlflow.proto.sqlflow_pb2_grpc as pb_grpc
 
-
 _LOGGER = logging.getLogger(__name__)
-
+handler = logging.StreamHandler(sys.stdout)
+_LOGGER.setLevel(logging.INFO)
+_LOGGER.addHandler(handler)
```
Comment (tonyyang-svail) [inline sqlflow/client.py]: I am not sure adding the log handler here is a good idea. Will those lines be executed every time we import the package?

@zou000 @ktong What do you think?

Comment (Yancey1989) [reply sqlflow/client.py]: Seems the Python interpreter would use a global table to cache the imported module, for the `import module` statement:
> if module is not imported python imports it, creates module object and puts newly created module object to the global table
if module is imported python just gets module object

ref: https://stackoverflow.com/questions/37067414/python-import-multiple-times/37077911

Comment (typhoonzero) [reply sqlflow/client.py]: You may clean all logging handlers before adding a new one every time to ensure that there are always only one handler global

Comment (Yancey1989) [reply sqlflow/client.py]: @typhoonzero  This would lead to re-initialize the logger when reloading the sqlflow magic command, seems it's not necessary?

**Pull request merged**